dispatch/apps/events/models.py

```diff
@@ -47,6 +47,8 @@ class Event(Model):
 
     is_published = BooleanField(default=False)
 
+    slug = SlugField(max_length=255)
```

Comment:
What happens if you just call `str(datetime.datetime.now() + datetime.timedelta(days=days))`?